File: DheerajKumar/problem16.cpp
/*16. WAP to take input a number (range 1 to 7) from the user and print the corresponding week day.
using if else 
using switch */

#include<iostream>
using namespace std;

void weekDayUsingIf(int num){
    if(num == 1){
        cout<<"-- Sunday --";
        return;
    }
    if(num == 2){
        cout<<"-- Monday --";
        return;
    }
    if(num == 3){
        cout<<"-- Thusday --";
        return;
    }
    if(num == 4){
        cout<<"-- Wednessday --";
        return;
    }
    if(num == 5){
        cout<<"-- Thursday --";
        return;
    }
    if(num == 6){
        cout<<"-- Friday --";
        return;
    }
    if(num == 7){
        cout<<"-- Saturday --";
        return;
    }

    cout<<"Out of range Input";
    return;
}


void weekDayUsingSwitch(int number){
    switch (number)
    {
    case 1:
        cout<<"-- Sunday --";
        return;
    case 2:
        cout<<" -- Monday --";
        return;
    case 3:
        cout<<"-- Thuesday --";
        return;
    case 4:
        cout<<"-- Wednessday --";
        return;
    case 5:
        cout<<"-- Thrusday --";
        return;
    case 6:
        cout<<"-- Friday --";
        return;
    case 7:
        cout<<"-- Saturday --";
        return;

    default:
        cout<<"Out of range Input ";
        return;
    }
}


int main(){
    int numberOfDay;
    cout<<"\n\nEnter the number -> ";
    cin>>numberOfDay;

    cout<<"\nThe day corresponding # ";
    weekDayUsingIf(numberOfDay);

    cout<<"\nThe day corresponding # ";
    weekDayUsingSwitch(numberOfDay);
}

File: Ashish Chauhan/problems-40/problem16.java
// 16.WAP to take input a number (range 1 to 7) from the user and print the corresponding week.
// using if else 
// using switch 
import java.util.Scanner;

public class WeekdayPrinter {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        System.out.print("Enter a number (1-7): ");
        int day = scanner.nextInt();
        
        // Using if-else
        if (day == 1) {
            System.out.println("Monday");
        } else if (day == 2) {
            System.out.println("Tuesday");
        } else if (day == 3) {
            System.out.println("Wednesday");
        } else if (day == 4) {
            System.out.println("Thursday");
        } else if (day == 5) {
            System.out.println("Friday");
        } else if (day == 6) {
            System.out.println("Saturday");
        } else if (day == 7) {
            System.out.println("Sunday");
        } else {
            System.out.println("Invalid input. Please enter a number between 1 and 7.");
        }
        
        // Using switch
        switch (day) {
            case 1: System.out.println("Monday"); break;
            case 2: System.out.println("Tuesday"); break;
            case 3: System.out.println("Wednesday"); break;
            case 4: System.out.println("Thursday"); break;
            case 5: System.out.println("Friday"); break;
            case 6: System.out.println("Saturday"); break;
            case 7: System.out.println("Sunday"); break;
            default: System.out.println("Invalid input. Please enter a number between 1 and 7.");
        }
        
        scanner.close();
    }
}
